### ISSUE TITLE
twisted.web.sux probably shouldn't be runnable

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4683]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)| @lvh reported|
|-|-|
|Trac ID|trac#4683|
|Type|task|
|Created|2010-10-11 23:25:05Z|

When cleaning up the source tree for stuff that still referenced doc/howto instead of doc/core/howto, I stumbled across t.w.sux.

http://twistedmatrix.com/trac/browser/tags/releases/twisted-8.1.0/twisted/web/sux.py#L639

I think that should be cleaned out. In fact I'm not sure why it's runnable at all. Can't we just remove the entire `if __name__ == '__main__':` block?

Attachments:

* [sux-main-4683.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/724/724d8d87068eb0357920fcb33cbd5269b23b6f18/ceb5474e22436bbd55e815545473ec28ad6ffdf9.patch) (819 bytes) - added by thijs on 2011-05-13 16:31:43Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4683 4683
type__task task
reporter__lvh lvh
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1286839505000000 1286839505000000
changetime__1305494726000000 1305494726000000
version__None None
owner__thijs thijs
cc__thijs cc__ivank
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="ivank's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|ivank set owner to @thijstriemstra|
|-|-|

1. This needs an empty news file `twisted/web/topfiles/4683.misc`

Otherwise, looks good for merging.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [31903]) Delete `twisted.web.sux` script support

Author: thijs
Reviewer: ivank
Fixes: [#4683](https://github.com/twisted/twisted/issues/4683)

Remove the self-check `__main__` code from `twisted.web.sux`.  The
module is tested by unit tests in `twisted.test.test_xml`.
